WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -81,15 +81,16 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v21.5-whats-new"
+msgctxt "v21.6-whats-new"
 msgid ""
-"We’ve been hard at work on extra-technical, behind-the-scenes fixes and updates to keep the Jetpack app running smoothly.\n"
+"Insight check—find out how your content is performing with our changes to Stats Insights and learn what types of content your readers like to see.\n"
```

Comment:
Does this read a little odd to you @mokagio? Or maybe I'm not sure what this is.

> Insight check—find out how your content

Reply:
Good point... 🤔 

It didn't read weird to me, but I can see how it could. The note would be as informative without that introduction:

```suggestion
"Find out how your content is performing with our changes to Stats Insights and learn what types of content your readers like to see.\n"
```

Reply:
I haven't gotten any feedback yet, so I'm making the call to accept the suggestion and remove `Insight check–`.

If that looked odd to someone inside the team working on the app, it might be confusing for users, too.